packages/usdk/packages/upstreet-agent/packages/react-agents/classes/generative-agent-object.ts

```diff
@@ -110,14 +111,28 @@ export class GenerativeAgentObject {
       });
     // });
   }
-  async evaluate(evaluator: Evaluator) {
-    return await this.conversation.typing(async () => {
+  async evaluate(evaluator: Evaluator, opts?: EvaluateOpts) {
+    const {
+      sendTyping = true,
```

Comment:
This seems like a micro-optimization; our "typing" has always be more of a "thinking" indicator. Not sure we should disable it even for the reactions case.

Reply:
was just thinking in the case of a lot reactions being made, the agent would just be appearing to type for a long time

Reply:
but yeah i agree this is to be done in a separate update/PR, so reverting this from here